website/source/docs/vagrantfile/winrm_settings.html.md

```diff
@@ -51,11 +51,13 @@ the Windows machine must be configured appropriately.
     **Note:** It is strongly recommended that you only use basic authentication for
     debugging purposes. Credentials will be transferred in plain text.
 
+* `config.winrm.execution_time_limit` (string) - The amount of time that is allowed to complete task. This defaults to "PT2H", that is 2 hours.
```

Comment:
Would you mind wrapping this line? It doesn't look like we are super strict about this, but most lines are wrapped to between 80-90 characters.